### PR TITLE
testcase_of _section_3.3

### PR DESCRIPTION
**Pull request by Ahad-04**
# Pull Request Template

## Prequisites

- [x] Did you run lint -cf `filename` on your changes, if you are commiting a .c file?
- [x] Did you run lint -sf `filename` on your changes, if you are commiting a .sh file?
- [x] Does your code compile properly using `make`?
- [x] Did you related issue number in the description?

## Description

<Write a short description of your changes>

## Technical Debt

<Write about any missing things that you are going to do in the future related to this PR>

### CODE REVIEW

File: code/src/test/test__sbi_get_impl_version/test__sbi_get_impl_version.c [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  uint64_t cause = read_csr(scause);
+
+  switch (cause) {
+    default:
+      break;
+  }
+}
+
+/**
+ * @brief Test case example checks for illegal instruction exception
```
Comment (mmhus) [inline code/src/test/test__sbi_get_impl_version/test__sbi_get_impl_version.c]: Update the test description

Comment (Ahad-04) [reply code/src/test/test__sbi_get_impl_version/test__sbi_get_impl_version.c]: Updated

File: testlists/test__sbi_get_impl_version.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+source export.sh
+make TEST_DIR="code/src/test/test__sbi_get_impl_id" CFLAGS="-DDEBUG"
```
Comment (mmhus) [inline testlists/test__sbi_get_impl_version.sh]: fix the path.

File: testlists/test__sbi_get_impl_id.sh [hunk 3]
```diff
@@ -2,5 +2,5 @@
 set -euo pipefail
 
 source export.sh
-make TEST_DIR="code/src/test/test__sbi_get_impl_id" CFLAGS="-DDEBUG"
+make TEST_DIR="code/src/test/test__sbi_get_impl_version" CFLAGS="-DDEBUG"
```
Comment (mmhus) [inline testlists/test__sbi_get_impl_id.sh]: fix the path.

File: code/src/test/test__sbi_get_impl_version/test__sbi_get_impl_version.c [hunk 4]
```diff
@@ -0,0 +1,35 @@
+#include "test_macros.h"
```
Comment (mmhus) [inline code/src/test/test__sbi_get_impl_version/test__sbi_get_impl_version.c:1]: Missing test

**Review by mmhus — APPROVED** [review]
ok

**Pull request merged**